### PR TITLE
Update emoji_test.go (fix errcheck issue)

### PR DESCRIPTION
**Pull request by TheInvincibleRalph**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28383

```release-note
NONE
```

### CODE REVIEW

File: server/channels/api4/emoji_test.go [hunk 1]
```diff
@@ -430,6 +430,7 @@ func TestDeleteEmoji(t *testing.T) {
 	th.AddPermissionToRole(model.PermissionDeleteOthersEmojis.Id, model.SystemUserRoleId)
 
 	client.Logout(context.Background())
```
Comment (hanzei) [inline server/channels/api4/emoji_test.go:432]: You need to assign the return value of `client.Logout` to a variable called `err` and then check it.

**Review by hanzei — CHANGES_REQUESTED** [review]

File: server/channels/api4/emoji_test.go (hunk 1 above)
Comment (hanzei) [reply server/channels/api4/emoji_test.go:432]: @TheInvincibleRalph Can you please push your changes? Thanks!

Comment (TheInvincibleRalph) [reply server/channels/api4/emoji_test.go]: Done. Thank you!

**Review by hanzei — COMMENTED** [review]
The code changes look good. 

Can you please update the golangci-lint config as part of this PR? Thanks!

**Comment by TheInvincibleRalph** [comment]
> The code changes look good.
> 
> Can you please update the golangci-lint config as part of this PR? Thanks!

Done, @hanzei

**Review by hanzei — APPROVED** [review]
Nice work :+1:

**Review by pvev — APPROVED** [review]
LGTM! thanks @TheInvincibleRalph 🚀

**Comment by hanzei** [comment]
@TheInvincibleRalph Can you please fix the merge conflict? Thanks.

**Comment by TheInvincibleRalph** [comment]
Done, @hanzei

**Pull request merged**